about.html

```diff
@@ -0,0 +1,190 @@
+        Explore topics in detail with an interactive learning experience on core subject matters in advanced batteries and electric & hybrid vehicles. <br>
+        Explore topics in detail with an interactive learning experience on core subject matters in advanced batteries and electric & hybrid vehicles.    
+        </p>    
+        <p class="text-white text-center">Meet battery manufacturers, suppliers, engineers, thought leaders and decision-makers for a conference and battery tech expo focused on the latest developments in the advanced battery and automotive industries.</p>
+
+        <h2 id="head-h2" class="text-white text-center">
+            Please contact us per Email for nay further questions<br>
+        </h2>
+            
+        <a href="" class="text-white">#EVTechExpo
+          <img class="p-1 d-none d-lg-inline-block"
+            src="./imgs/icons/ArrowLeftToRight.svg" alt="arrow left to right" />
+        </a>
+            
+        
+        </h2>
+      </div>
+
+    </section>
+    
+    <section id="headline2" 
+    class="w-100 text-center pt-5 pb-5">
+        <h2 id="head2-h2">Join Peers at the Advanced Battery and H/EV Technology Community</h2>
+        <div id="head2-line"></div>
+
+        <p id="head2-p1" class="d-none d-lg-block m-0">Join more than 770+ suppliers to showcase products representing the entire supply chain. This is the only trade fair where you can connect with over 10,000+ attendees in 3 days.This is the only trade fair where you can connect with over 10,000+ attendees in 3 days.</p>
+        <p id="head2-p2" class="m-0 mb-5">Join more than 770+ suppliers to showcase products representing the entire supply chain. </p>
+
+        <img width="80%" src="./imgs/electric-battery-logo2.png" alt="logo">  
+
+    </section>
+
+    <hr class="d-md-none">
+
+    <section id="past-conference" 
+    class="w-100 text-center">
+      <h5><b>See the past Event Summits</b></h5>
+      <div id="past-line"></div>
+      <p id="past-p" >have a look at EU's most important event in batteries and EV tech industries.</p>
+
+      <div id="video-container" class="row row-cols-1 row-cols-md-2 justify-content-between">
+        <div class="col">
+          <iframe width="100%" height="100%" src="https://www.youtube.com/embed/WhBTyulrktc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen
+          ></iframe>
+        </div>
+
+        <div class="col">
+          <iframe width="100%" height="100%" src="https://www.youtube.com/embed/L4bwJAt9iC8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </div>
+      </div>
```

Comment:
- [ ] I like how you replaced the images with `iframes`, it's a great look. However, for now, please follow the original template. Kindly make sure this section has two images with centered texts in the middle.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/b2f628b1-1668-4a9c-83d2-74697ca42259) | ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/a0ddfd14-63b6-4b59-9853-1ffeea36e138) |